packages/internal/bridge/sdk/src/lib/tenderly.ts

```diff
@@ -22,13 +29,41 @@ import { getTenderlyEndpoint } from './utils';
  *    }
  *  }
  */
+export function unwrapStateObjects(
+  stateObjects: StateObject[],
+): Record<string, Record<string, Record<string, string>>> {
```

Comment:
Alternatively, we could have a `StateObjects` class which essentially combines the storage of `StateObject` array and this `unwrap` function. LMK thoughts.